docs/Splashkit/Applications/npm-installation-guide.md

```diff
@@ -0,0 +1,153 @@
+# Npm installation guide: Windows and Mac
+
+###### (scroll down for Mac instructions)
+
+### What does npm do?
+
+Npm is used to check markdown files (md) to match Prettier (plug-in) and needs Node.js to run properly.
+
+## Windows Guide
+
+---
```

Comment:
**suggestion:** I think we can drop this. The header format already adds a horizontal line underneath the text (see rendered [markdown](https://github.com/thoth-tech/documentation/blob/2a282896604e5299f46a056759602beab9a56a36/docs/Splashkit/Applications/npm-installation-guide.md#windows-guide) in GitHub).